### PR TITLE
Adjust image diff threshold for testMayaLights

### PR DESCRIPTION
**Pull request by debloip-adsk**

### CODE REVIEW

**Review by lilike-adsk — COMMENTED** [review]
is this for MacOS specifically? I'm tweaking the threshold for another test case,

**Comment by debloip-adsk** [comment]
Yes for macOS specifically, it's causing failures in preflights e.g. for #54, I've checked the output images and it does match the expected result so it looks like just a threshold issue

**Comment by lilike-adsk** [comment]
> Yes for macOS specifically, it's causing failures in preflights e.g. for #54, I've checked the output images and it does match the expected result so it looks like just a threshold issue

If that, could we disable the test on MacOS first? I found if we use a pretty large threshold, we might miss the real failures. Meanwhile, we need to investigate why MacOS produced different images than Win/Linux for most test cases.

File: test/lib/mayaUsd/render/mayaToHydra/testMayaLights.py [hunk 1]
```diff
@@ -20,12 +20,23 @@
...
+import platform
+
 class TestMayaLights(mtohUtils.MtohTestCase): #Subclassing mtohUtils.MtohTestCase to be able to call self.assertSnapshotClose
     # MayaHydraBaseTestCase.setUpClass requirement.
     _file = __file__
 
-    IMAGE_DIFF_FAIL_THRESHOLD = 0.01
-    IMAGE_DIFF_FAIL_PERCENT = 0.2
+    @property
+    def imageDiffFailThreshold():
```
Comment (lilike-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/testMayaLights.py]: great, miss "self" as parameter?

Comment (debloip-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/testMayaLights.py]: Yes definitely, though somehow it ran fine locally without it. Thanks for catching it

**Review by lilike-adsk — APPROVED** [review]

**Pull request merged**